plugins/uefi/fu-uefi-device.c

```diff
@@ -725,6 +725,7 @@ static void
 fu_uefi_device_init (FuUefiDevice *self)
 {
 	fu_device_set_protocol (FU_DEVICE (self), "org.uefi.capsule");
+	fu_device_set_version_format (FU_DEVICE (self), FWUPD_VERSION_FORMAT_TRIPLET);
```

Comment:
So as part of of this change, shouldn't the default move to PLAIN or UNKNOWN for UEFI?  And then all metadata updates for the right value?